colorama/initialise.py

```diff
@@ -49,10 +55,13 @@ def init(autoreset=False, convert=None, strip=None, wrap=True):
 
 
 def deinit():
-    if orig_stdout is not None:
+    global orig_stdout, orig_stderr
+    if orig_stdout is not UNSET:
         sys.stdout = orig_stdout
-    if orig_stderr is not None:
+        orig_stdout == UNSET
```

Comment:
I think this statement has no effect (did you mean `=`?)

Reply:
good catch, oopsie!

Reply:
and, amusingly, fixing that assignment breaks a bunch of other tests. Thinking...

Reply:
Was a result of tests not cleaning up properly on tearDown. Fixed now.